xml/src/test/scala/akka/stream/alpakka/xml/ClasspathResources.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.xml
+
+import java.io.InputStream
+
+import akka.stream.IOResult
+import akka.stream.scaladsl.{ Source, StreamConverters }
+import akka.util.ByteString
+
+import scala.concurrent.Future
+import scala.io.{ Source => ScalaIOSource }
+
+trait ClasspathResources {
```

Comment:
how is character encoding handled in those? I think we should always be explicit with character encoding and never user platform defaults.